testing/scenario_app/android/README.md

```diff
@@ -16,23 +16,46 @@ $ ./testing/scenario_app/run_android_tests.sh android_debug_unopt
 $ ./testing/scenario_app/run_android_tests.sh android_debug_unopt_arm64
 ```
 
+## Debugging
+
+Debugging the tests on CI is not straightforward but is being improved:
+
+- <https://github.com/flutter/flutter/issues/143458>
+- <https://github.com/flutter/flutter/issues/143459>
+
+Locally (or on a temporary PR for CI), you can run the tests with the
+`--smoke-test` argument to run a single test by class name, which can be useful
+to verify the setup:
+
+```sh
+# From the root of the engine repository
+$ ./testing/scenario_app/run_android_tests.sh android_debug_unopt_arm64 --smoke-test dev.flutter.scenarios.EngineLaunchE2ETest
+```
+
+The result of `adb logcat` and screenshots taken during the test will be stored
+in a logs directory, which is either `FLUTTER_LOGS_DIR` (on CI) or locally in
+`out/.../scenario_app/logs`.
+
 ## CI Configuration
 
 See [`ci/builders/linux_android_emulator.json`](../../../ci/builders/linux_android_emulator.json)
 , and grep for `run_android_tests.sh`.
 
 The following matrix of configurations is tested on the CI:
 
-| API Version | Graphics Backend    | Skia Gold                                                        | Rationale                                                  |
-| ----------- | ------------------- | ---------------------------------------------------------------- | ---------------------------------------------------------- |
+<!-- TODO(matanlurey): Blocked by https://github.com/flutter/flutter/issues/143471.
 | 28          | Skia                | [Android 28 + Skia][skia-gold-skia-28]                           | Older Android devices (without `ImageReader`) on Skia.     |
 | 28          | Impeller (OpenGLES) | [Android 28 + Impeller OpenGLES][skia-gold-impeller-opengles-28] | Older Android devices (without `ImageReader`) on Impeller. |
-| 34          | Skia                | [Android 34 + Skia][skia-gold-skia-34]                           | Newer Android devices on Skia.                             |
-| 34          | Impeller (OpenGLES) | [Android 34 + Impeller OpenGLES][skia-gold-impeller-opengles-34] | Newer Android devices on Impeller with OpenGLES.           |
-| 34          | Impeller (Vulkan)   | [Android 34 + Impeller Vulkan][skia-gold-impeller-vulkan-34]     | Newer Android devices on Impeller.                         |
-
 [skia-gold-skia-28]: https://flutter-engine-gold.skia.org/search?left_filter=AndroidAPILevel%3D28%26GraphicsBackend%3Dskia&negative=true&positive=true&right_filter=AndroidAPILevel%3D28%26GraphicsBackend%3Dskia
 [skia-gold-impeller-opengles-28]: https://flutter-engine-gold.skia.org/search?left_filter=AndroidAPILevel%3D28%26GraphicsBackend%3Dimpeller-opengles&negative=true&positive=true&right_filter=AndroidAPILevel%3D28%26GraphicsBackend%3Dimpeller-opengles
+-->
+
```

Comment:
Did you mean to move this? if so ignore this comment.

Reply:
Yeah unfortunately markdown tables break with inline comments. Was intentional.